<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>studentdashboard</class>
 <widget class="QDialog" name="studentdashboard">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>559</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color:rgb(230, 230, 250)</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>70</y>
     <width>101</width>
     <height>311</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QPushButton" name="homeButton">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(114, 14, 158);
color: white</string>
      </property>
      <property name="text">
       <string>HOME</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="assignmentButton">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(114, 14, 158);
color: white</string>
      </property>
      <property name="text">
       <string>ASSIGNMENT</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="internalButton">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(114, 14, 158);
color: white</string>
      </property>
      <property name="text">
       <string>INTERNAL</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="logOutButton">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(114, 14, 158);
color: white</string>
      </property>
      <property name="text">
       <string>LOG OUT</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>20</y>
     <width>561</width>
     <height>421</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(221, 160, 221);
color: black</string>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="Home">
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>140</y>
       <width>191</width>
       <height>91</height>
      </rect>
     </property>
     <property name="text">
      <string>Home</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="Assignment">
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>180</y>
       <width>191</width>
       <height>91</height>
      </rect>
     </property>
     <property name="text">
      <string>Assignment</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="Internal">
    <widget class="QTableView" name="tableView">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>531</width>
       <height>361</height>
      </rect>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
